userspace/falco/app/actions/helpers_inspector.cpp

```diff
@@ -20,30 +20,28 @@ limitations under the License.
 #include <fcntl.h>
 
 #include <plugin_manager.h>
+#include <configuration.h>
 
 #include "helpers.h"
 
 #ifdef _WIN32
 #define PATH_MAX 260
 #endif
 
-/* DEPRECATED: we will remove it in Falco 0.34. */
-#define FALCO_BPF_ENV_VARIABLE "FALCO_BPF_PROBE"
-
 using namespace falco::app;
 using namespace falco::app::actions;
 
 falco::app::run_result falco::app::actions::open_offline_inspector(falco::app::state& s)
 {
 	try
 	{
-		s.offline_inspector->open_savefile(s.options.trace_filename);
-		falco_logger::log(falco_logger::level::INFO, "Reading system call events from file: " + s.options.trace_filename + "\n");
+		s.offline_inspector->open_savefile(s.config->m_replay.m_capture_file);
+		falco_logger::log(falco_logger::level::INFO, "Reading system call events from file: " + s.config->m_replay.m_capture_file + "\n");
```

Comment:
```suggestion
		falco_logger::log(falco_logger::level::INFO, "Replaying events from the capture file: " + s.config->m_replay.m_capture_file + "\n");
```

N.B. _capture files_ are not tied to system call